_shortlinks/home.md

```diff
@@ -1,4 +1,5 @@
 ---
 redirect: https://isabelle.in.tum.de
 description: The official website and download page.
+category: Infrastructure
```

Comment:
Why is this `Infrastructure` and not `Resources`?

Reply:
Since the documentation is linked separately as resource and the home-page mainly serves as download site.

Reply:
I would have said that the download website is more like a resource (it gives something to the user) than being part of the infrastructure (it provides something to developers). Or how would you frame the two categories?